platform/broadcom/sonic-platform-modules-dell/s5224f/scripts/platform_sensors.py

```diff
@@ -119,8 +122,11 @@ def print_temperature_sensors():
     print ('  CPU Temp:                       ',\
         (get_pmc_register('CPU_temp')))
 
-ret_status, ipmi_cmd_ret = subprocess.getstatusoutput('echo 0 > /sys/module/ipmi_si/parameters/kipmid_max_busy_us')
-if ret_status:
+file = '/sys/module/ipmi_si/parameters/kipmid_max_busy_us'
+if os.path.exists(file):
+    with open(file, 'w') as f:
+        f.write('0\n')
```

Comment:
There are multiple ways to fail the block, for example, could not open file, or write failure.